src/dune_rules/odoc.ml

```diff
@@ -668,20 +673,46 @@ let check_mlds_no_dupes ~pkg ~mlds =
       ]
 ;;
 
+let report_warnings warnings =
+  match warnings with
+  | [] -> ()
+  | _ :: _ ->
+    let l =
+      warnings
+      |> List.map ~f:(fun (mld : Doc_sources.mld) -> Path.Local.to_string mld.in_doc)
```

Comment:
I am not sure this can ever happen (or whether there's something that fixes the ordering in the list) but maybe it could be useful to sort the strings so the files are always reported in the same order.

Makes it a bit easier to use in tests as well.